### PR TITLE
Quimicita Shuttle - Chemistry for One

### PR DESCRIPTION
**Pull request by kennedytcooper**
## About the PR

IT'S TIME TO COOK! A rework of the Bocadillo food truck for a lone chemist to craft and sell meds. Costs just a bit more than the food version. Does not come with a chem dispenser but it's ready to house one, does come with a vending machine for chem jugs to get you started.

**Media**

Screenshot from 2023-07-31 17-30-39

**Changelog**

:cl:
- add: the Quimicita a rework of the food truck for a different kind of cooking (chemistry)

### CODE REVIEW

**Comment by potato1234x** [comment]
finally i can make chem bombs without spending 17 quadrillion dollars

**Comment by Cheackraze** [comment]
I'm concerned about this ship just being purchased to be looted round start but im willing to test it

**Comment by Cheackraze** [comment]
Also it appears you have the wrong shuttle file here

File: Resources/Maps/Shuttles/quimicita.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  postmapinit: true
+tilemap: {}
+entities:
+- proto: CableApcExtension
+  entities:
+  - uid: 1
+    components:
+    - pos: 2.5,-25.5
+      type: Transform
+...
```
Comment (Cheackraze) [inline Resources/Maps/Shuttles/quimicita.yml:12]: Awfully small for a grid file

**Review by Cheackraze — CHANGES_REQUESTED** [review]

**Comment by hord-brayden** [comment]
> I'm concerned about this ship just being purchased to be looted round start but im willing to test it

I agree, I think there's a lot of great items on here that are accessible at a super low price point. I would assume the chemmaster is the biggest concern and frankly, thematically, I don't think it fits. Hear me out:

Typically most of the crew upstream don't have access to chemmaster and usually resort to a more crude method of mixing and concocting their brews. My suggestion, and it's just that - a suggestion - we ditch the chemmaster, the chemvend, and the hot plate. Additionally replace with the following:

-Start with all 3 of the Chem crates (D,S,P)
-beakers and chem equipment galore (cryo beakers, droppers, etc)
-a set or two of chem analysis goggles
-Ditch the hot plate for a microwave since it can achieve the same effect but it's way sketchier and loud/more useful
-add a condiment dispenser
-welding fuel tank
-add a basic weapon(mk52, viper, makeshift laser pistol)
-ditch the drain in the floor for toilets and plungers
-maybe 1 hydro tray, with just ambrosia and cannabis seeds or something 

Here's some 'maybe fun' options:
-add fun wardrobe options like gas masks, bowler hats, glasses, Jensen coats, sunglasses, machete, nitrile/latex gloves, gas analyzer (whatever makes you chuckle the most)
-add various wooden barricades around the ship in the extemporaneous airlocks
-lots of curtains around to obscure vision inside
-add a camera server and monitor, along with some external cameras so people can cook with peace of mind

Just throwing some ideas out there, as I would love something like this. Even if we took the price to be below the bocadillo, and nerfed this one where it has 1 APU, but supplemented with a jr pacman that need to be run periodically, and a less demanding power infrastructure?

**Comment by Cheackraze** [comment]
feel free to re-open if you want to come back to it

**Pull request closed without merging**